plugins/smart_proxy_remote_execution_ssh/control

```diff
@@ -11,7 +11,8 @@ Package: ruby-smart-proxy-remote-execution-ssh
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.11.0~rc3),
-  ruby-foreman-remote-execution-core (>= 1.0.0),
-  ruby-smart-proxy-dynflow (>= 0.1.0), ruby-smart-proxy-dynflow (<< 1.0.0), ruby-net-ssh
+  ruby-smart-proxy-dynflow (>= 0.5.0), ruby-smart-proxy-dynflow (<< 1.0.0), ruby-net-ssh
+Breaks: ruby-foreman-remote-execution-core (<<1.5.0)
+Replaces: ruby-foreman-remote-execution-core (<<1.5.0)
```

Comment:
1.5.0 is the version that this PR is removing, so `<<` seems wrong? Maybe `<<1.5.1`?